moped-editor/src/queries/signals.js

```diff
@@ -6,7 +6,7 @@ export const SIGNAL_PROJECTS_QUERY = gql`
   query SignalProjectsQuery {
     moped_project(
       where: {
-        is_retired: { _neq: true }
+        is_deleted: { _eq: false }
```

Comment:
Interesting how you flipped the operator and the operand for a net non-change, but I think this is more readable. Great catch.